azure-keyvault/azure/keyvault/secrets/_client.py

```diff
@@ -81,18 +95,26 @@ def vault_url(self):
         return self._vault_url
 
     def get_secret(self, name, version, **kwargs):
-        """Get a specified from the vault.
+        """Get a specified secret from the vault.
 
         The GET operation is applicable to any secret stored in Azure Key
         Vault. This operation requires the secrets/get permission.
 
         :param str name: The name of the secret.
-        :param str version: The version of the secret.  If not specified the latest version of
+        :param str version: The version of the secret. If not specified the latest version of
```

Comment:
We should clarify "not specified". Something like "If version is None or the empty string, ..."